### DISCUSSION TITLE
Spline Spark Agent Roadmap - timelines for release supporting RDD actions

### DISCUSSION THREAD

**Discussion by rkrumins** [AbsaOSS/spline-spark-agent#519] [Q&A] [answered]
Hi all,

I have been using Spline Spark Agent for quite some time and it works wonderfully for capturing the lineage of various Spark ETL jobs. I noticed that there is a milestone for the 1.0.0 release (https://github.com/AbsaOSS/spline-spark-agent/milestone/10) and I am really interested in finding out when that would be coming out. The feature I am most excited about is RDD Lineage support - specifically the functionality addressed by https://github.com/AbsaOSS/spline-spark-agent/issues/33 issue. 

Would it be possible to find out approximately a release of Spline Spark Agent supporting RDD actions would be available? Any pointers would be much appreciated.

**Comment by wajda** [comment, marked as answer]
It's pretty much ready, we just want to align the agent release with the server release which still lacks of some important features. We expect it ti be ready in some few weeks.

**Reply by rkrumins** [reply]
That is great news @wajda - looking forwards to testing that out 💯

**Reply by wajda** [reply]
You can try it already. Download the latest agent bundle JAR from our CI server - https://teamcity.jetbrains.com/project/OpenSourceProjects_AbsaOSS_SplineAgentSpark?branch=%3Cdefault%3E&mode=builds

**Reply by cerveada** [reply]
Please note that #33 is an epic (ticket grouping multiple sub-tickets) and only one part of it is currently planed for 1.0.0 (#498)

**Reply by wajda** [reply]
[1.0.0 version](https://github.com/AbsaOSS/spline-spark-agent/releases/tag/release%2F1.0.0) is released. Please check it out.
